src/components/FImage.vue

```diff
@@ -124,6 +125,8 @@ const props = withDefaults(defineProps<FImageProps>(), {
   backgroundColor: '',
 });
 
+registerIcons('icons', { landscape });
+
```

Comment:
What's the advantage with previous implem? This looks more custom and maybe more complex to apprehend.